app/scripts/modules/core/src/widgets/cluster/ClusterMatches.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { IClusterMatch } from './clusterMatches.component';
+import { NgReact } from 'core/reactShims/ngReact';
+
+export interface IClusterMatchesProps {
+  matches: IClusterMatch[]
+}
+
+export class ClusterMatches extends React.Component<IClusterMatchesProps> {
+
+  public render() {
+    const { matches } = this.props;
+    const { AccountTag } = NgReact;
+    if (!matches || !matches.length) {
+      return (<div>(no matches)</div>);
+    }
+    return (
+      <div>
```

Comment:
ul li ?

Reply:
.nostyle on ul removes all paddings/margins and list styles.

Reply:
Makes more sense - will change